### ISSUE TITLE
use 2to3 is invalue

### ISSUE THREAD

**Issue by sama2689** [nsh87/regressors#24]
When installing to Python3.8 on conda I keep getting the error that use_2to3 is invalid. Why is this and is there a fix? I have updated setuptools in conda but can't quite figure out what the issue is.

```
ERROR: Command errored out with exit status 1:
     command: 'C:\Users\USER\Anaconda3\envs\dvh38\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\USER\\AppData\\Local\\Temp\\pip-install-nfs30n22\\regressors_0fe7e4fd6596496ab4d564996ff24a0c\\setup.py'"'"'; __file__='"'"'C:\\Users\\USER\\AppData\\Local\\Temp\\pip-install-nfs30n22\\regressors_0fe7e4fd6596496ab4d564996ff24a0c\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\USER\AppData\Local\Temp\pip-pip-egg-info-apx50gdw'
         cwd: C:\Users\USER\AppData\Local\Temp\pip-install-nfs30n22\regressors_0fe7e4fd6596496ab4d564996ff24a0c\
    Complete output (1 lines):
    error in regressors setup command: use_2to3 is invalid.
```

**Comment by YaoWang-2020** [comment]
Is this issued fixed? I still get the same error : use_2to3 is invalid.

**Comment by aguptaz** [comment]
After removing  '**use_2to3**' , i got same issue

**Comment by jingjtang** [comment]
Hi， I also got `error in regressors setup command: use_2to3 is invalid.` when using `pip install regressors`

**Comment by xfry-lixuan** [comment]
I also got error in regressors setup command: use_2to3 is invalid. when using pip install regressors

**Comment by xfry-lixuan** [comment]
is this issued fixed?

**Comment by sama2689** [comment]
Yes the issue is fixed. Please see the thread linked by bastula

**Comment by nsh87** [comment]
Sorry, I did not realize this was still getting use. I am mostly focused on large language modeling these days, however I fixed the 2to3 issue. Please see: https://github.com/nsh87/regressors/issues/26#issuecomment-1370293417

*tl;dr*: install with `pip` using GitHub commit directly for Python 3.10, etc.
